zstacklib/zstacklib/utils/sanlock.py

```
@@ -373,13 +418,55 @@ def direct_dump(path, offset, length):
+    with lock.NamedLock("check-vglk-{}-lease".format(vg_name)):
+        sector_size = get_sector_size(vg_name)
+        offset = VGLK_BEGIN * sector_size
+        r, o, e = bash.bash_roe("sanlock direct read -r lvm_{0}:{1}:/dev/mapper/{0}-lvmlock:{2}"
+                                .format(vg_name, "VGLK", offset))
+        init = False
+        for line in (o+e).split().splitlines():
+            line = line.strip()
+            if line.startswith("read done "):
+                read_rv = int(line.split()[-1])
+                if lease_corrupted_err(read_rv):
+                    logger.debug("vglk lease corrupted, err %s, reinitialize it directly.".format(o+e))
```

Ruff (0.11.9)

`.format` call has unused arguments at position(s): 0

Remove extra positional arguments at position(s): 0

(F523)